### DISCUSSION TITLE
How to use vanillajs in a php environment using web3modal v2

### DISCUSSION THREAD

**Discussion by C0INON** [WalletConnect/walletconnect-monorepo#2484] [Developer Support]
I was using web3modal v1 version using php and vanillajs on the Apache server. For v2, you should only use node, but you need an example that is available in php on the Apache server.
If you have an example file, please let me know

**Comment by xzilja** [comment]
For now easiest way would be to take this package `https://docs.walletconnect.com/2.0/web/web3modal/html/sign-api/installation` and build a commonjs file using something like https://vitejs.dev so you can use it in a `<script />` tag.

We'll be working on providing this out of the box at some point as well

**Comment by finessevanes** [comment]
@C0INON we're working on the OOTB solution. You can follow it [here](https://github.com/WalletConnect/web3modal/issues/678).
